app/views/idv/in_person/verify_info/show.html.erb

```diff
@@ -25,11 +25,9 @@ locals:
 <div class='margin-top-4 margin-bottom-2'>
   <div class="grid-row grid-gap grid-gap-2 padding-bottom-1 border-bottom border-primary-light">
     <dl class="grid-col-fill margin-y-0">
-      <% if @capture_secondary_id_enabled %>
         <div class="padding-y-1">
           <h2 class="h4 margin-y-0"><%= t('headings.state_id') %></h2>
         </div>
```

Comment:
ditto re: unindenting because the conditionals are no longer there
```suggestion
      <div class="padding-y-1">
        <h2 class="h4 margin-y-0"><%= t('headings.state_id') %></h2>
      </div>
```